### DISCUSSION TITLE
Get value of another field in update operation

### DISCUSSION THREAD

**Discussion by bn-piar** [prisma/prisma#15392] [Q&A] [answered]
Hey guys. 

I have a model `Stand` (like in convenience store) that has fields `fillState` and `maxCapacity`. `fillState` is derived using the following arithmetics: 
```javscript
// products is something I receive from somewhere
const fillState = (products.length / maxCapacity) * 100
```
I often receive new products and I need to keep `fillState` valid for which I need to know what is `maxCapacity` for current stand is. 

My code 

```javascript
const updated = prisma.stand.update({ where: { standId }, data: {
  fillState: Math.round((products.length / maxCapacity) * 100)  /* how do I get maxCapacity value here? */
});
```

Just a silly example of a feature that I'm interested in.

**Comment by austincrim** [comment, marked as answer]
Hey @bn-piar 👋,

You should take a look at the [`fieldReference` preview feature](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#compare-columns-in-the-same-table). If `maxCapacity` is in the `Stand` model, that query should be possible with the new feature.

**Comment by johnmangan** [comment]
I'd love to be mistaken, but it seems this is only available for find/select queries, not updates. https://github.com/prisma/prisma/issues/19442

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
